Gemfile.lock

```diff
@@ -255,6 +260,7 @@ PLATFORMS
   x86_64-linux
```

Comment:
Would love to talk about this more, should Taija and I delete our 'darwin' out of our platform?  Do we need ruby here?